canopy/design-system/README.md

```diff
@@ -1,7 +1,9 @@
-This project was bootstrapped with
-[Create React App](https://github.com/facebook/create-react-app).
+# Canopy Design System
```

Comment:
Can you update the commit message to make it clear you are updating the canopy design system readme. This make it sound like you are updating the top level splinter readme

Reply:
updated